### PR TITLE
rename sample files once unzipped

### PR DESCRIPTION
**Pull request by NathanPJF**
### What are you trying to accomplish with this PR?

Companion piece to this PR in `slate`: https://github.com/Shopify/slate/pull/179

The idea: After running `slate theme`, you are given an unzipped `slate-src.zip` plus two files `config-sample` and `.gitignore-sample`.  Let's save the manual effort of renaming both these files when you start off.

### Checklist
For contributors:
- [ ] I have updated the documentation in the [README file](https://github.com/Shopify/slate-cli/blob/master/README.md) to reflect these changes, if applicable.

For maintainers:
- [ ] I have :tophat:'d these changes.
- [ ] I have bumped the `package.json` version in a separate PR, if applicable.

Considerations:
- [ ] These changes will require the [public Slate docs](https://shopify.github.io/slate/) to be updated.  See the [Shopify/slate repo](https://github.com/Shopify/slate/blob/master/CONTRIBUTING.md#documentation) for notes on updating docs, if applicable.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -77,6 +77,20 @@ export function writePackageJsonSync(target, name = 'theme') {
 }
 
 /**
+ * Rename a file path
+ *
+ * @param {string} current - The path to the file currently.
+ * @param {string} target - The renamed path of the file.
+ */
+export function renameFile(current, target) {
+  rename(current, target, (err) => {
```
Comment (chrisberthe) [inline src/utils.js]: Should we be wrapping this in `return new Promise((resolve, reject) => {});`?

File: src/utils.js [hunk 2]
```diff
@@ -77,6 +77,20 @@ export function writePackageJsonSync(target, name = 'theme') {
...
+    if (err) {
+      throw err;
```
Comment (chrisberthe) [inline src/utils.js]: If so, `reject(err)` here. And `resolve()` outside of the condition.

**Review by chrisberthe — CHANGES_REQUESTED** [review]

**Comment by NathanPJF** [comment]
New project... that fyre. 🔥

File: src/commands/theme.js [hunk 3]
```diff
@@ -89,7 +89,6 @@ export default function(program) {
           ];
 
           function renamePromiseFactory({oldFile, newFile, dir}) {
-            console.log(`  Renaming ${oldFile} to ${newFile}...`);
```
Comment (NathanPJF) [inline src/commands/theme.js]: Decided I didn't like the message after it said "you're done".

Comment (chrisberthe) [inline src/commands/theme.js]: Good call removing this.

File: src/commands/theme.js [hunk 4]
```diff
@@ -81,6 +81,28 @@ export default function(program) {
...
           return null;
         })
+        .then(async () => {
+
+          const sampleFiles = [
+            {oldFile: 'config-sample.yml', newFile: 'config.yml', dir: root},
+            {oldFile: '.gitignore-sample', newFile: '.gitignore', dir: root},
+          ];
+
+          function renamePromiseFactory({oldFile, newFile, dir}) {
```
Comment (chrisberthe) [inline src/commands/theme.js]: Thanks for this ES6 lesson 🥇

**Review by chrisberthe — APPROVED** [review]

**Pull request merged**